storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1445,9 +1463,10 @@ bool Redo_Log_Data_Manager::stop_at(lsn_t lsn, lsn_t checkpoint_lsn) {
   archived_log_monitor.stop();
 
   /* to ensure redo logs are not disabled during the backup, reopen the log
-  files to read HEADER */
-  if (!reopen_log_files(lsn)) {
-    xb::error() << "Cannot find file with LSN " << lsn;
+  files to read HEADER. */
+  if (opt_lock_ddl == false && archived_log_state == ARCHIVED_LOG_NONE &&
```

Comment:
what about archived_log_state == ARCHIVED_LOG_MATCHED. We are still copying from live redo.

Reply:
ARCHIVED_LOG_MATCHED. Server would ensure redo are not disabled, So PXB is safte